app/controllers/service_requests_controller.rb

```diff
@@ -132,8 +132,8 @@ def review
     @review       = true
     @portal       = false
     @admin        = false
-    @merged       = false
-    @consolidated = true
+    @merged       = true
+    @consolidated = false
```

Comment:
Bug: Review uses the merged calendar, i.e. the "Calendar Tab" calendar, not the consolidated calendar.